app/helpers/login.ts

```diff
@@ -0,0 +1,25 @@
+import { Keypair } from "stellar-sdk";
+
+function getPublicKey(secretKey: string): string {
+  try {
+    return Keypair.fromSecret(secretKey).publicKey();
+  } catch (error) {
+    const errorTag = document.querySelector("#error");
```

Comment:
This is breaking the reason why we're using React. We should make this using the tools that React give us.

Reply:
I removed that logic and added a React state to handle the error message rendering.